src/convertToObject.js

```diff
@@ -13,6 +13,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const stringToArr = sourceString.split(':').join(';').split(';')
+    .map(item => item.trim());
+  const cssRules = {};
+
+  for (let i = 0; i < stringToArr.length; i++) {
+    if (stringToArr[i] !== '') {
+      cssRules[stringToArr[i]] = stringToArr[i + 1];
+      i++;
+    }
```

Comment:
let's create array of arrays in the first place, so you don't need to use indexes that much

let's try to create such array out of the initial string
[[property, value], [property, value], etc]